package.json

```diff
@@ -28,23 +29,34 @@
     "prettier": "^1.17.1",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
+    "react-scripts": "3.2.0",
     "sass-loader": "^7.1.0",
     "style-loader": "^0.23.1",
-    "webpack": "^4.30.0",
+    "webpack": "^4.41.0",
```

Comment:
Since `webpack` is a transitive dependency of `react-scripts` (i.e. Create React App), I get the following error when I run `npm start`:

```
There might be a problem with the project dependency tree.
It is likely not a bug in Create React App, but something you need to fix locally.

The react-scripts package provided by Create React App requires a dependency:

  "webpack": "4.41.0"

Don't try to install it manually: your package manager does it automatically.
However, a different version of webpack was detected higher up in the tree:

  /Users/coka/work/srce-desktop-app/node_modules/webpack (version: 4.41.2) 

Manually installing incompatible versions is known to cause hard-to-debug issues.
```

What this means is that this line should be removed. We should rely on CRA to handle all webpack-related configuration. If you run into issues after removing this line, try deleting `package-lock.json`, and running `npm install` again.

Reply:
It worked for me with webpack 4.41.0, old version threw an error, but I didn't try with both versions, I will try now.

Reply:
Without webpack in package.json, I get this error: 
![ss](https://user-images.githubusercontent.com/48222323/69233343-062b0b80-0b8d-11ea-838d-59d239a02d25.png)

Reply:
Ok, I get it. Both package-lock.json and node_modules required to delete.
Problem solved.